### DISCUSSION TITLE
Island Chat

### DISCUSSION THREAD

**Discussion by Evalutik** [BG-Software-LLC/SuperiorSkyblock2#1786] [Approved Feature Requests] [answered]
I would like players on the same island to be able to have the /isc command so that they can write to the island's chat. If a player is on his own island, then when he writes a message to the chat, it automatically enters the chat of his island and the participants of this island see it. If the player is not on the island, then nothing happens to the message. To write to the island chat while not on the island, you need to write /isc <message> I would also like the island chat to be stirred up with the AdvancedBan plugin (https://www.spigotmc.org/resources/advancedban.8695/)

Thx

**Comment by Lemvs** [comment, marked as answer]
Implemented as /island localchat in latest dev build.

@OmerBenGera you can mark this answered and close it.
